flow-typed/react-reconciler.js

```diff
@@ -60,175 +60,175 @@ declare module 'react-reconciler' {
-    ): void,
-    commitTextUpdate(textInstance: TI, oldText: string, newText: string): void,
-    resetTextContent(instance: I): void,
-    appendChild(parentInstance: I, child: I | TI): void,
-    appendChildToContainer(container: C, child: I | TI): void,
-    insertBefore(parentInstance: I, child: I | TI, beforeChild: I | TI): void,
-    insertInContainerBefore(
-      container: C,
-      child: I | TI,
-      beforeChild: I | TI,
-    ): void,
-    removeChild(parentInstance: I, child: I | TI): void,
-    removeChildFromContainer(container: C, child: I | TI): void,
-  };
-
-  declare type PersistentUpdatesHostConfig<T, P, I, TI, C, CC, PL> = {
-    cloneInstance(
-      instance: I,
-      updatePayload: null | PL,
-      type: T,
-      oldProps: P,
-      newProps: P,
-      internalInstanceHandle: OpaqueHandle,
-      keepChildren: boolean,
-      recyclableInstance: I,
-    ): I,
-
-    createContainerChildSet(container: C): CC,
-
-    appendChildToContainerChildSet(childSet: CC, child: I | TI): void,
-    finalizeContainerChildren(container: C, newChildren: CC): void,
-
-    replaceContainerChildren(container: C, newChildren: CC): void,
-  };
-
-  declare type HydrationHostConfig<T, P, I, TI, HI, C, CX, PL> = {
-    // Optional hydration
-    canHydrateInstance(instance: HI, type: T, props: P): null | I,
-    canHydrateTextInstance(instance: HI, text: string): null | TI,
-    getNextHydratableSibling(instance: I | TI | HI): null | HI,
-    getFirstHydratableChild(parentInstance: I | C): null | HI,
-    hydrateInstance(
-      instance: I,
-      type: T,
-      props: P,
-      rootContainerInstance: C,
-      hostContext: CX,
-      internalInstanceHandle: OpaqueHandle,
-    ): null | PL,
-    hydrateTextInstance(
-      textInstance: TI,
-      text: string,
-      internalInstanceHandle: OpaqueHandle,
-    ): boolean,
-    didNotMatchHydratedContainerTextInstance(
-      parentContainer: C,
-      textInstance: TI,
-      text: string,
-    ): void,
-    didNotMatchHydratedTextInstance(
-      parentType: T,
-      parentProps: P,
-      parentInstance: I,
-      textInstance: TI,
-      text: string,
-    ): void,
-    didNotHydrateContainerInstance(parentContainer: C, instance: I | TI): void,
-    didNotHydrateInstance(
-      parentType: T,
-      parentProps: P,
-      parentInstance: I,
-      instance: I | TI,
-    ): void,
-    didNotFindHydratableContainerInstance(
-      parentContainer: C,
-      type: T,
-      props: P,
-    ): void,
-    didNotFindHydratableContainerTextInstance(
-      parentContainer: C,
-      text: string,
-    ): void,
-    didNotFindHydratableInstance(
-      parentType: T,
-      parentProps: P,
-      parentInstance: I,
-      type: T,
-      props: P,
-    ): void,
-    didNotFindHydratableTextInstance(
-      parentType: T,
-      parentProps: P,
-      parentInstance: I,
-      text: string,
-    ): void,
-  };
+  // declare export type HostConfig<T, P, I, TI, HI, PI, C, CC, CX, PL> = {
```

Comment:
why are they commented out?

Reply:
Those types, particularly ssr and mutation blocks need to be flattened.